set.mm

```diff
@@ -147154,6 +147148,398 @@ of an open range of nonnegative integers (of length equal to the length of
+    IXJXNRWFWIWNXMXGWIWLXLAOWIWJXKWDWHWDWHLMVFWIVGVHVIVJVKVNVLXEWRTDWSTDXHWTRXE
+    CQVMAQVOTWRWSVPVQVRVSVTVR $.
+
+  ${
+    $d N y $.  $d V x $.  $d x y $.
+    $( Sets of words are disjoint if each set contains extensions of distinct
+       words of a fixed length.  (Contributed by AV, 29-Jul-2018.)  (Revised by
+       AV, 6-May-2020.) $)
+    disjwrdpfx $p |- Disj_ y e. W { x e. Word V | ( x prefix N ) = y } $=
+      ( cword cv cpfx co invdisjrab ) ABEDFAGCHIJ $.
+  $}
+
+  ${
+    $d x A $.  $d x S $.  $d x Y $.  $d x Z $.
+    $( Joining a prefix with an adjacent subword makes a longer prefix.
+       (Contributed by AV, 7-May-2020.) $)
+    ccatpfx $p |- ( ( S e. Word A /\ Y e. ( 0 ... Z )
+                       /\ Z e. ( 0 ... ( # ` S ) ) )
+        -> ( ( S prefix Y ) ++ ( S substr <. Y , Z >. ) ) = ( S prefix Z ) ) $=
+      ( wcel cc0 cfz co chash cfzo syl2anc caddc wceq cmin wa zcnd eqtrd adantr
+      cfv syl3anc vx cword w3a cop csubstr cconcat wfn wf pfxcl 3ad2ant1 swrdcl
+      cpfx ccatcl wrdf 3syl ccatlen simp1 fzass4 biimpri simpld 3adant1 swrdlen
+      pfxlen oveq12d cc cz elfzelz ad2antrl 3impb ad2antll pncan3d oveq2d mpbid
+      ffn fneq2d pfxfn 3adant2 cv wo 3ad2ant2 fzospliti eleq2d biimpar ccatval1
+      simpr ccatval2 anim1i ancomd fzosubel syl wb subidd eqcomd oveq1d eqeltrd
+      mpbird swrdfv syldan elfzoelz adantl npcand fveq2d 3eqtrd jaodan eqfnfvd
+      pfxfv simpl3 eqtr4d ) BAUBZEZCFDGHEZDFBISZGHZEZUCZUAFDJHZBCULHZBCDUDUEHZU
+      FHZBDULHZXOXSFXSISZJHZUGZXSXPUGXOXSXIEZYBAXSUHYCXOXQXIEZXRXIEZYDXJXKYEXNA
+      BCUIUJZXJXKYFXNABCDUKUJZAXQXRUMKAXSUNYBAXSVNUOXOYBXPXSXOYADFJXOYAXQISZXRI
+      SZLHZDXOYEYFYAYKMYGYHAXQXRUPKXOYKCDCNHZLHDXOYICYJYLLXOXJCXMEZYICMXJXKXNUQ
+      ZXKXNYMXJXKXNOZYMDCXLGHEZYMYPOYOFCDXLURUSUTVAZABCVCKZABCDVBVDXOCDXJXKXNCV
+      EEZXJYOOZCXKCVFEZXJXNCFDVGZVHPVIZXJXKXNDVEEYTDXNDVFEXJXKDFXLVGVJPVIVKQZQV
+      LVOVMXJXNXTXPUGXKBDAVPVQXOUAVRZXPEZOZUUEXSSZUUEBSZUUEXTSZXOUUFUUEFCJHZEZU
+      UECDJHZEZVSZUUHUUIMZUUGUUFUUAUUOXOUUFWEZXOUUAUUFXKXJUUAXNUUBVTZRUUEFDCWAK
+      XOUULUUPUUNXOUULOZUUHUUEXQSZUUIUUSYEYFUUEFYIJHZEZUUHUUTMXOYEUULYGRXOYFUUL
+      YHRXOUVBUULXOUVAUUKUUEXOYICFJYRVLWBWCAXQXRUUEWDTUUSXJYMUULUUTUUIMXOXJUULY
+      NRXOYMUULYQRXOUULWEUUECABXFTQXOUUNOZUUHUUEYINHZXRSZUVDCLHZBSZUUIUVCYEYFUU
+      EYIYKJHZEZUUHUVEMXOYEUUNYGRXOYFUUNYHRXOUVIUUNXOUVHUUMUUEXOYICYKDJYRUUDVDW
+      BWCAXQXRUUEWFTXOUUNUVDFYLJHZEUVEUVGMUVCUVDUUECNHZUVJXOUVDUVKMUUNXOYICUUEN
+      YRVLRZUVCUVKUVJEZUVKCCNHZYLJHZEZUVCUUNUUAOUVPUVCUUAUUNXOUUAUUNUURWGWHUUEC
+      DCWIWJXOUVMUVPWKUUNXOUVJUVOUVKXOFUVNYLJXKXJFUVNMXNXKUVNFXKCXKCUUBPWLWMVTW
+      NWBRWPWOABCDUVDWQWRUVCUVFUUEBUVCUVFUVKCLHUUEUVCUVDUVKCLUVLWNUVCUUECUUNUUE
+      VEEXOUUNUUEUUECDWSPWTXOYSUUNUUCRXAQXBXCXDWRUUGXJXNUUFUUJUUIMXOXJUUFYNRXJX
+      KXNUUFXGUUQUUEDABXFTXHXE $.
+  $}
+
+  ${
+    $d k B $.  $d k S $.  $d k T $.
+    $( Recover the left half of a concatenated word.  (Contributed by Mario
```

Comment:
Maybe:
The left "concatenand" of a concatenated word is the prefix of that word of its own length.

Reply:
The comment was written by Mario, I  do not want to change it without his acceptance. In Addition, I think the word "concatenand" is not commonly used.